pkg/controller/migmigration/migrate.go

```diff
@@ -96,6 +96,13 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (bool, e
 		if err != nil {
 			return false, err
 		}
+		if !migration.Spec.Stage {
+			plan.SetClosed()
+			err = r.Update(context.TODO(), plan)
+			if err != nil {
+				return false, err
```

Comment:
For a second I thought this would fail to requeue since we're returning `false, err`, but I checked the calling block and looks like it will do some further error processing for us. Recalled that returning `true, err` would have triggered a time delayed requeue for cases where we won't get another event.

But think this will work :+1: